src/commands/environments/update.ts

```diff
@@ -21,21 +21,21 @@ export default class UpdateEnvironment extends UpdateCommand<CreateEnvironmentPa
 
     public async run(): Promise<void> {
         await this.requireProject()
-        const { environment } = await inquirer.prompt([environmentPrompt], {
+        const { _environment } = await inquirer.prompt([environmentPrompt], {
             token: this.authToken,
             projectKey: this.projectKey
         })
 
         this.writer.blankLine()
         this.writer.statusMessage('Current values:')
-        this.writer.statusMessage(JSON.stringify(environment, null, 2))
+        this.writer.statusMessage(JSON.stringify(_environment, null, 2))
```

Comment:
oh, based on the variable update method it looks like that should be logging the current object properties

Reply:
updated the environmentPrompt to return the whole `environment` instead of just the id

Reply:
Nice! This may be getting out of scope of this ticket, but it logs _all_ the properties of the environment, and in this context it may make sense to only log the values of the editable properties

Reply:
I'll create another ticket to tackle this👍🏻